src/zzz_od/application/shiyu_defense/shiyu_defense_app.py

```
@@ -86,57 +85,82 @@ def wait_loading(self) -> OperationRoundResult:
+                        return self.round_success(ShiyuDefenseApp.STATUS_ALL_FINISHED)
+                elif len(digits) == 1:
+                    # 只有一个数字，可能是 "5"，说明已完成
+                    total = int(digits[0])
+                    log.info('剧变节点进度 已完成 %d', total)
+                    self.config.critical_max_node_idx = total
+                    return self.round_success(ShiyuDefenseApp.STATUS_ALL_FINISHED)
+                else:
+                    log.info('OCR 进度解析失败: %s', progress_text)
+                    next_idx = 1
+        else:
+            log.info('未读到剧变节点进度，尝试从 run_record 获取')
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)